### ISSUE TITLE
ssdb-dump reports negative number of entries

### ISSUE THREAD

**Issue by ignaciovazquez** [ideawu/ssdb#615]
Hello there,

While I'm trying to dump the database, I'm getting the following output. 

Is everything OK with this dump? Is there a limit to the number of entries we can have in SSDB and SSDB dumps? (that looks like an unsigned 32-bit int)

Thanks,

Ignacio

```
ssdb (cli) - ssdb command line tool.
Copyright (c) 2012-2014 ssdb.io

'h' or 'help' for help, 'q' to quit.

ssdb-server version: 1.6.8.8

ssdb 127.0.0.1:8888> ok
(0.000 sec)
ssdb 127.0.0.1:8888> ssdb-dump - SSDB backup command
Copyright (c) 2012-2014 ssdb.io

recv begin...
received 1 entry(s)
received 10 entry(s)
received 100 entry(s)
received 1000 entry(s)
received 10000 entry(s)
received 100000 entry(s)
received 1000000 entry(s)
received 10000000 entry(s)
received 100000000 entry(s)
received 1000000000 entry(s)
received -2147483648 entry(s)
received -2142045490 entry(s)
```

**Comment by ideawu** [comment]
Hi, the backup data is OK, it is only the problem of the printed message, I will fix it.

**Comment by ideawu** [comment]
Hi, this error is fixed on master and stable-1.8.0

**Comment by ignaciovazquez** [comment]
Thanks!

Ignacio
